docs/src/tutorials/inequality.md

```diff
@@ -0,0 +1,29 @@
+# Solve BVP with Inequality Boundary Conditions
+
+When dealing with scenarios that boundary conditions are stronger than just explicit values at specific points—such as inequality boundary conditions, where the solution must satisfy constraints like staying within upper and lower bounds—we need a more flexible approach. In such cases, we can use the `maxsol` and `minsol` functions provided by BoundaryValueDiffEq.jl to specify such inequality conditions.
```

Comment:
This case isn't doing inequalities, it's making the maximum equal to a value.

Reply:
Yeah, thanks for the elaboration! That makes sense, maxsol/minsol puts more constraints than inequality conditions here, and maxsol/minsol should only focus on the maximum/minimum value satisfying a specific value.